explorer/main.cpp

```diff
@@ -80,6 +80,20 @@ auto ExplorerMain(int argc, char** argv, void* static_for_main_addr,
           clEnumValN(ProgramPhase::All, "trace_all",
                      "Include trace output for all phases.")));
 
+  cl::list<FileContext> allowed_file_contexts(
+      cl::desc("Select file contexts for which you want to include the trace "
+               "output"),
+      cl::values(
+          clEnumValN(
+              FileContext::Main, "file_main",
+              "Include trace output for file containing the main function"),
+          clEnumValN(FileContext::Prelude, "file_prelude",
+                     "Include trace output for prelude"),
+          clEnumValN(FileContext::Import, "file_import",
+                     "Include trace output for imports"),
+          clEnumValN(FileContext::All, "file_all",
+                     "Include trace output for all files")));
```

Comment:
These flag names should include `trace` -- `-trace_file_main`, `-trace_file_prelude`, etc maybe?

Reply:
Yes, it would make more sense. But I also think flag names are getting bigger, should we create short aliases as well?

Reply:
I think it'd be useful to add a "trace everything" flag that turns on tracing for all files and all steps, for debugging purposes. (But let's do that as a separate change.)